src/backend/joanie/core/models/products.py

```diff
@@ -362,6 +430,12 @@ class OrderCourseRelation(models.Model):
         related_name="order_relations",
         on_delete=models.RESTRICT,
     )
+    course_runs = models.ManyToManyField(
```

Comment:
I'm skeptical that we need this relation... My understanding is that we could refer to the course runs linked on the product course relation model... 
Indeed, for courses it is important to copy it here because it is what the user buys, but for the course run, I don't see any problem to propose course runs as they are defined on the product at the moment the user tries to enroll. WDYT?

Reply:
Agreed :+1: :muscle: